kvui.py

```diff
@@ -708,8 +708,10 @@ def _handle_player_id(self, node: JSONMessagePart):
             text = f"Game: {slot_info.game}<br>" \
                    f"Type: {SlotType(slot_info.type).name}"
             if slot_info.group_members:
-                text += f"<br>Members:<br> " + \
-                        "<br> ".join(self.ctx.player_names[player] for player in slot_info.group_members)
+                text += f"<br>Members:<br> " + "<br> ".join(
+                    self.ctx.player_names[player].replace("[", "&bl;").replace("]", "&br;")
```

Comment:
```suggestion
                    escape_markup(self.ctx.player_names[player])
```

Reply:
Thanks for pointing out this function exists!